database/models/InviteCode.js

```diff
@@ -0,0 +1,48 @@
+const { DataTypes, UUIDV4 } = require('sequelize');
+var RandExp = require('randexp');
+
+module.exports = (sequelize) => {
+  const InviteCode = sequelize.define(
+    'InviteCode',
+    {
+      id: {
+        type: DataTypes.UUID,
+        defaultValue: UUIDV4,
+        allowNull: false,
+        primaryKey: true,
+      },
+      userId: {
+        type: DataTypes.UUID,
+        allowNull: false,
+      },
+      code: {
+        type: DataTypes.STRING,
+        defaultValue: () => new RandExp(/^[a-zA-Z0-9]{8}$/).gen(),
```

Comment:
I think we should go with a package called [short-unique-id](https://www.npmjs.com/package/short-unique-id) as we already discussed on [discord](https://discord.com/channels/774749371568357406/791234295657267220/884567981261217873).